doc/source/train/distributed-pytorch/reporting-results.rst

```diff
@@ -0,0 +1,63 @@
+Reporting results to Ray Train
```

Comment:
I am thinking maybe we want this User Guide to be about "Monitoring and logging metrics", which might be more intuitive for users. This should cover the basics of:

1. Reporting
2. Logging

From here, we can link to:
1. Checkpointing
2. Experiment Tracking